docs/transactions/transaction-log/README.md

```diff
@@ -1,10 +1,30 @@
 ---
 description: >-
   A Transaction Log is a record of a MobileCoin transaction that was constructed
-  and sent from this wallet, or that was received at an address belonging to an
```

Comment:
Are we no longer creating Transaction Logs for received TXOs?

Reply:
Correct, transaction logs as a concept were reworked in a previous PR and there is not a separation between sent transactions and received (and potentially spent) txos